src/js/elements/mapchip/components/snackbar_content.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import { SnackbarFilterLableAvatar } from '../../../styled_components/mapchip/snackbar';
+
+import './mapchip.module.css';
```

Comment:
I'd expect `mapchip.module.css` to be imported in `mapchip.js` not this module. If the CSS applies to the snackbar_content then possibly rename the CSS file.